apps/web-giddh/src/app/ledger/ledger.vm.ts

```
@@ -152,9 +152,7 @@ export class LedgerVM {
                 bl.particular = bl.selectedAccount ? bl.selectedAccount?.uniqueName || bl.selectedAccount?.value : bl.particular;
                 bl.isInclusiveTax = false;
                 // filter taxes uniqueNames
-                bl.taxes = [...bl.taxesVm?.filter(p => p.isChecked).map(p => p?.uniqueName)];
-                // filter discount
-                bl.discounts = bl.discounts?.filter(p => p.discountValue && p.isActive);
+                bl.taxes = [...bl.taxesVm?.map(p => p?.uniqueName)];
```

Biome (2.3.13)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)